README.md

```diff
@@ -47,6 +47,7 @@ Additional options that can be passed through include:
 | `language` | The ISO 2 digit language code | "en"|
 | `input.placeholderText` | The placeholder text for the input | "Start typing an address"|
 | `input.applyFocus` | Whether to apply focus to the search field | true|
+| `input.applyFocus` | Whether to apply focus to the search field | true|
```

Comment:
This is a duplication of the line above. Did you intend to document the new `countryCodeMapping` property instead?

Reply:
I removed the line. 
Thanks for the catch